app/models/ability.rb

```diff
@@ -44,7 +44,7 @@ def not_signed_in
 
     # can view Commercials of confirmed Events
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
-    can :show, User
+    can [:show, :create], User
```

Comment:
> Where would that authentication happen (ie where do we access the create action of user)?

[here](https://github.com/openSUSE/osem/pull/907/files#diff-8c13e9cfe1f5edf7d4923e633a08f790R37):
```Ruby
  def create
    unless current_user
       @user = User.new(user_params)
       authorize! :create, @user  # <- this line
        if @user.save
          sign_in(@user)
        else
         # this path would have no authorization
         flash[:error] = "Could not save user: #{@user.errors.full_messages.join(', ')}"
         render action: 'new'
         return
       end
     end
     # more code
  end
```
cancancan was complaining that no authorization was performed in that path. It is either this or `skip_authorization_check`